Measurements/Tests.qs

```diff
@@ -378,6 +389,22 @@ namespace Quantum.Kata.Measurements {
 
         CheckSuperpositionBitstringsMeasurement(5, [30,14,10,7],  // [11110,01110,01010,00111]
                                                    [1,17,21,27]); // [00001,10001,10101,11011]
+
+        CheckSuperpositionBitstringsMeasurement(2, [2,1],       // [10,01]
+                                                   [1]); // [01]
+
+        CheckSuperpositionBitstringsMeasurement(2, [1,2],       // [01,10]
+                                                   [1]); // [01]
+
+        CheckSuperpositionBitstringsMeasurement(3, [7,5],       // [111,101]
+                                                   [2]); // [010]
+
+        CheckSuperpositionBitstringsMeasurement(4, [13,11,7,3], // [1101,1011,0111,0011]
+                                                   [5,2]); // [0101,0010]
+
+        CheckSuperpositionBitstringsMeasurement(5, [14,17,9], // [01110,10001,01001]
```

Comment:
The testing harness doesn't support preparation of superposition states with 3 terms in the superposition (you can see this in `StatePrep_BitstringSuperposition` operation which does nothing if Length(bits) is anything but 1, 2 or 4). This is a good point - we need to improve the test harness either to cover other lenths or to fail early instead of providing the user invalid inputs. 

Long-term we might want to look into using [StatePreparationPositiveCoefficients](https://docs.microsoft.com/qsharp/api/qsharp/microsoft.quantum.preparation.statepreparationpositivecoefficients) library operation to prepare the superposition states instead of writing the cases by hand.

For now, could you please delete this test and add the following line to the beginning of `StatePrep_BitstringSuperposition` operation (line 256, right after variable L is defined):

    Fact(L == 1 or L == 2 or L == 4, "State preparation only supports arrays of 1, 2 or 4 bit strings");

Reply:
done